CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## master / unreleased
 
 * [CHANGE] Querier / ruler: Change `-querier.max-fetched-chunks-per-query` configuration to limit to maximum number of chunks that can be fetched in a single query. The number of chunks fetched by ingesters AND long-term storare combined should not exceed the value configured on `-querier.max-fetched-chunks-per-query`. #4260
+* [CHANGE] Add timeout for waiting on compactor to become ACTIVE in the ring. #4262
```

Comment:
[nit] We use CHANGE for breaking changes. This is more an enhancement.

```suggestion
* [ENHANCEMENT] Add timeout for waiting on compactor to become ACTIVE in the ring. #4262
```

Reply:
Fixed! Thanks for the review!